docusaurus/docs/how-to-guides/app-plugins/implement-rbac-in-app-plugins.md

```diff
@@ -14,8 +14,28 @@ keywords:
   - access control
 ---
 
+
+
+
 Role-based access control (RBAC) in Grafana app plugins is essential for creating secure and tailored user experiences. By implementing RBAC, you ensure that sensitive functionalities and data are only accessible to users with appropriate permissions, enhancing both security and usability. Proper configuration is crucial as misconfigurations can lead to security vulnerabilities.
 
+## Table of Contents
+
+- [Before you begin](#before-you-begin)
+- [Defining roles](#defining-roles)
+- [Actionsets, extend actions with permissions](#actionsets-extend-actions-with-permissions)
+  - [Grafana Action Sets](#grafana-action-sets)
+  - [Limitations](#limitations)
+  - [Codepaths](#codepaths)
+  - [Plugin Action Sets Definition](#plugin-action-sets-definition)
+  - [Practical Example](#practical-example)
+- [Secure frontend includes](#secure-frontend-includes)
+- [Secure proxied routes](#secure-proxied-routes)
+- [Secure backend resources](#secure-backend-resources)
+- [Implementation](#implementation)
+- [Implement frontend access control checks](#implement-frontend-access-control-checks)
+- [Assigning roles](#assigning-roles)
+
```

Comment:
Please remove this. I don't believe we have a TOC anywhere else in documentation except the auto generated one in the right hand column which should be plenty as it's position is also fixed so always available when scrolling.

![image](https://github.com/user-attachments/assets/b77b7b70-68c5-4965-9531-f889065e4a40)